core/node/house_keeper/README.md

```diff
@@ -0,0 +1,11 @@
+# zkSync Era housekeeper
+
+This crate contains functionality for performing “administrative” work to keep the system flowing. It does:
+
+- reporting
+- archiving
+- retrying
+- monitoring
```

Comment:
Nit: I feel that "monitoring" and "reporting" refer to roughly the same thing. Also, "retrying" and "queue cleaning" items are not clear; are they about about jobs (if so, it could make sense to have a single item: "Scheduling jobs, retrying failed jobs, and cleaning obsolete jobs").

Reply:
now should be correct